src/odemis/driver/xt_client.py

```diff
@@ -1194,6 +1193,9 @@ def _doMoveRel(self, future, shift):
         # Check range (for the axes we are moving)
         for an in shift.keys():
             rng = self.axes[an].range
+            if rng == (0, 2 * math.pi):
```

Comment:
I'm a little bit picky, but let's add `and an in ("rx", "rz")`. Otherwise, if for some (very unlikely) reason a linear axis has limits exactly between (0 -> 2 pi), it'll start behaving strange.